### DISCUSSION TITLE
Writing about the loading model

### DISCUSSION THREAD

**Discussion by ingale726** [fastapi/fastapi#9227] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from pydantic import BaseModel
from fastapi import FastAPI,Form,Body
from starlette.requests import Request
from starlette.staticfiles import StaticFiles
from starlette.templating import Jinja2Templates
from aioify import aioify
app = FastAPI()
app.mount("/static", StaticFiles(directory="static"), name="static")
templates = Jinja2Templates(directory="templates")
from gensim.models.fasttext import FastText
import random
import time

from pymilvus import (
    connections,
    utility,
    FieldSchema,
    CollectionSchema,
    DataType,
    Collection,
)
import numpy as np

connections.connect("default", host="172.16.75.171", port="19530")

hello_milvus = Collection("long_128")
model = FastText.load('fasttext.model')


# get --------------------------------
@app.get("/")
def hello(request: Request):#
    return templates.TemplateResponse('upload.html',{"request": request})
# ------------------------------------

# post ------------------------------------------------------


@app.post("/post")#
def search(request: Request,query: str = Body(...)):
    query_data = query
    start_time = time.time()
    embedding = model.wv[query]
    normalized_v = (embedding / np.sqrt(np.sum(embedding ** 2))).tolist()
    search_params = {
        "metric_type": "IP",
        "params": {"ef": 250},
    }
    results = hello_milvus.search(
        [normalized_v],
        anns_field="embeddings",
        param=search_params,
        limit=200,
        output_fields=["string"],
        expr=None,
        consistency_level="Strong"
    )
```

### Description

When I put the load model outside the function, the concurrency is very low, how do I write this line of code?

`model = FastText.load('fasttext.model')`

Or how do I load the model to provide inference services?

The way this code is written is fine in Flask; it can achieve 100 concurrent amounts

### Operating System

Linux

### FastAPI Version

0.92.0

### Python Version

3.7

**Comment by yinziyan1206** [comment]
use threadpool to execute `model.wv[query]`

**Reply by ingale726** [reply]
Ok, let me have a try
